MANIFEST.in

```diff
@@ -28,7 +30,6 @@ global-exclude *.odt
 global-exclude *.orc
 global-exclude *.sas7bdat
 global-exclude *.sav
-global-exclude *.so
```

Comment:
Right now this allows the .so to make it into the wheel build as part of setuptools, but unfortunately also includes as in the sdist. Open item that needs to be addressed but figured this was easiest to do for now to continue CI